src/main/java/Chauncey/ui/Chauncey.java

```diff
@@ -0,0 +1,200 @@
+                }
+                if (eventDetails.length > 3) {
+                    throw new ChaunceyException("Task details more than expected! Please only input task description, task start time and task end time.");
+                }
+                String startTime = eventDetails[eventDetails.length - 2].trim();
+                String endTime = eventDetails[eventDetails.length - 1].trim();
+                tasks[numOfTask] = new Event(eventDetails[0].trim(), startTime, endTime);
+                break;
+            default:
+                System.out.println("Invalid task type.");
+            }
+            numOfTask++;
+            System.out.println("Got it. I've added this task: ");
+            tasks[numOfTask - 1].outputTaskDetails();
+            System.out.println("Now you have " + numOfTask + " tasks in the list.");
+        } catch (ChaunceyException e) {
+            System.out.println("Error: " + e.getMessage());
+        }
+    }
+
+    private static void removeTask(String command) {
+        try {
+            if (numOfTask == 0) {
+                throw new ChaunceyException("There is no task in the list. Can't do remove command.");
+            }
+            int taskNumber = getTaskNumber(command);
+            String taskDetails = tasks[taskNumber - 1].getTaskDetails();
+            for (int i = taskNumber - 1; i < numOfTask - 1; i++) {
+                tasks[i] = tasks[i + 1];
+            }
+            numOfTask--;
+            System.out.println("removed: " + taskDetails);
+        } catch (ChaunceyException e) {
+            System.out.println("Error: " + e.getMessage());
+        }
+    }
+
+    private static void listTasks() {
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 1; i<= numOfTask; i++) {
+            System.out.print(i + ".");
+            tasks[i-1].outputTaskDetails();
+        }
+    }
+
+    private static void markTask(String command) {
+        try {
+            if (numOfTask == 0) {
+                throw new ChaunceyException("There is no task in the list. Can't do mark command.");
+            }
+            int taskNumber = getTaskNumber(command);
+            if (tasks[taskNumber-1].getStatus()) {
+                throw new ChaunceyException("Task " + taskNumber + " is already marked done.");
+            }
+            tasks[taskNumber - 1].markAsDone();
+            System.out.println("Nice! I've marked this task as done:");
+            tasks[taskNumber - 1].outputTaskDetails();
+        } catch (ChaunceyException e) {
+            System.out.println("Error: " + e.getMessage());
+        }
+    }
+
+    private static void unmarkTask(String command) {
+        try {
+            if (numOfTask == 0) {
+                throw new ChaunceyException("There is no task in the list. Can't do unmark command.");
+            }
+            int taskNumber = getTaskNumber(command);
+            if (!tasks[taskNumber-1].getStatus()) {
```

Comment:
Do make sure arithmetics have a space between the symbols just as how you have written in other parts of the code
```suggestion
if (!tasks[taskNumber - 1].getStatus()) {
```